.travis.yml

```diff
@@ -21,3 +21,13 @@ before_install:
 
 script:
     - sudo ./automation/run-tests.sh
+
+after_success:
+    - sudo pip install python-coveralls
```

Comment:
We used `coveralls` in the tox, can we keep using the same?

Can we just do this using tox? Not critical, but it will look cleaner IMO.

Reply:
Also we are currently using `coveralls` and not `python-coveralls` in tox. Do we need to change the implementation?

Reply:
Changed to tox. Done.

Reply:
Using the tox for coveralls will need a long package list of ubuntu for compiling gobject as tox will try to install nmstate before run coveralls. Not sure why.

Reply:
[`skip_install=true`](https://tox.readthedocs.io/en/latest/config.html#conf-skip_install) should allow to avoid installing nmstate in the virtual env.